validation/cubed_sphere_eddying_aquaplanet/cubed_sphere_eddying_aquaplanet.jl

```diff
@@ -0,0 +1,232 @@
+
+    u_bcs_faces = Tuple(
+        inject_cubed_sphere_exchange_boundary_conditions(
+            FieldBoundaryConditions(face_grid, (Face, Center, Center); top=FluxBoundaryCondition(τx.data.faces[face_number][1:Nx+1, 1:Ny, 1])),
+            face_number,
+            grid.face_connectivity
+        )
+        for (face_number, face_grid) in enumerate(grid.faces)
+    )
+
+    v_bcs_faces = Tuple(
+        inject_cubed_sphere_exchange_boundary_conditions(
+            FieldBoundaryConditions(face_grid, (Center, Face, Center); top=FluxBoundaryCondition(τy.data.faces[face_number][1:Nx, 1:Ny+1, 1])),
+            face_number,
+            grid.face_connectivity
+        )
+        for (face_number, face_grid) in enumerate(grid.faces)
+    )
+
+    u_bcs = CubedSphereFaces{typeof(u_bcs_faces[1]), typeof(u_bcs_faces)}(u_bcs_faces)
+    v_bcs = CubedSphereFaces{typeof(v_bcs_faces[1]), typeof(v_bcs_faces)}(v_bcs_faces)
+
+    return u_bcs, v_bcs
+end
+
+#####
+##### Script starts here
+#####
+
+function cubed_sphere_eddying_aquaplanet(grid_filepath)
+
+    ## Grid setup
+
+    H = 8kilometers
+    grid = ConformalCubedSphereGrid(grid_filepath, Nz=1, z=(-H, 0))
+
+    ## "Tradewind-like" zonal wind stress pattern where -π/2 ≤ φ ≤ π/2
+    τx(φ) = 0.1 * exp(-10 * (φ - π/3)^2) + 0.1 * exp(-10 * (φ + π/3)^2) - 0.2 * exp(-8φ^2) + 0.19 * exp(-20φ^2)
+
+    coriolis = HydrostaticSphericalCoriolis(scheme=VectorInvariantEnstrophyConserving())
+    R = grid.faces[1].radius
+    Ω = coriolis.rotation_rate
+
+    # Streamfunction integrated using Mathematica
+    # Scale by Ω^2 * R^2 * sin(φ) to get units of momentum flux (m²/s²) on the sphere.
+    # Minus sign because a negative flux drives currents in the positive x-direction.
+    # Hmmm, I think I messed up the magnitude but the resulting wind stress patterns should do the trick for now. It's a little too strong (stronger than the 0.1 N/m² we expect from τx).
+    # I'm further scaling it down by a factor of 0.1 so the wind stress is even weaker (for stability/debugging).
+    ψ̃(λ, φ) = - 0.1 * Ω^2 * R^2 * sin(φ) * (-0.02802495608 * erf(1.054092553 * (π - 3φ)) - 0.06266570687 * erf(2.828427125φ) + 0.03765160933 * erf(4.472135955φ) + 0.02802495608 * erf(1.054092553 * (π + 3φ)))
+
+    λ̃(λ) = (λ + 180)/ 360 * 2π
+    φ̃(φ) = φ / 180 * π
+    ψ(λ, φ) = ψ̃(λ̃(λ), φ̃(φ))
+
+    u_top_flux, v_top_flux, ψ₀ = diagnose_velocities_from_streamfunction(ψ, grid)
+
+    u_bcs, v_bcs = cubed_sphere_surface_momentum_flux_bcs(u_top_flux, v_top_flux, grid)
+
+    ## Model setup
+
+    model = HydrostaticFreeSurfaceModel(
+               architecture = CPU(),
+                       grid = grid,
+         momentum_advection = nothing,
+               free_surface = ExplicitFreeSurface(gravitational_acceleration=0.1),
+                   coriolis = coriolis,
+        boundary_conditions = (u=u_bcs, v=v_bcs),
+                    closure = HorizontallyCurvilinearAnisotropicDiffusivity(νh=1000),
```

Comment:
Can also try

```julia
closure = HorizontallyCurvilinearAnisotropicBiharmonicDiffusivity(νh=νh4)
```

Not sure a reasonable value for `νh4` though 🤷

Reply:
True. I just took a guess at `νh=1000`. Will definitely give biharmonic a try. Movies might be prettier.